python/tvm/micro/contrib/zephyr.py

```diff
@@ -703,6 +709,21 @@ def write(self, data, timeout_sec):
             raise TransportClosedError()
         return self.fd_transport.write(data, timeout_sec)
 
+    def _qemu_check_stdout(self):
+        for line in self.proc.stdout:
+            line = str(line)
+            _LOG.debug(line)
+            if "[QEMU] CPU" in line:
+                self._queue.put(self._qemu_ready_msg)
+                return
```

Comment:
don't return here--doing so could freeze the subprocess. instead, continue to read the output so that the pipe is drained.